packages/eas-cli/src/credentials/android/credentials.ts

```diff
@@ -8,9 +9,13 @@ export interface Keystore {
   keystore: string;
   keystorePassword: string;
   keyAlias: string;
-  keyPassword: string;
+  keyPassword?: string;
```

Comment:
Making this field optional seems risky to me. Have you tested what happens when you try to build (with EAS Build) an android project with a keystore that has an empty key password? 
(cc @wkozyra95)

Reply:
it's possible that this might not work. I have that on my todo list to test that. with EAS Builds. 

Is this affect classic builds? If someone would update credentials using eas and later build the same app with expo-cli would it use those new credentials?

Reply:
So there is a concept of 'legacy credentials' in Android, when credentials are only keyed under the project and not the `package`. Right now `expo-cli` and `eas-cli` are all using the legacy credentials because I haven't built out the methods to use the new credentials strucutre in cli yet 😅  

I was planning on making anyone who created new Keystores in `eas-cli` to use the new credentials structure. If they went back to `expo-cli`, it only uses legacy credentials so this shouldnt be a problem

Reply:
Also there are currently a bunch of keystores in our db that dont have `keyPassword` for classic builds - i had migrated them over and checked they were indeed valid.  If you were already gonna do it, would really appreciate if you could test the key password stuff on turtle.

Reply:
Thanks for the clarification 👍

Reply:
Tested p12 file on EAS
 - if empty string is passed it won't pass validation
 - if random value is passed build will file during signing
 - if the same password is passed as keystorePasowrd and keyPassword it works
 
 You can go with the third approach as a temporary solution or wait a bit with merging this PR until API changes on our side will be ready